src/rules/at-conditional-rule-no-parentheses/index.js

```diff
@@ -0,0 +1,70 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+import _ from "lodash";
+
+export const ruleName = namespace("at-conditional-rule-no-parentheses");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Unexpected () used to surround statements for @-rules"
+});
+
+// postcss picks up else-if as else.
+const conditional_rules = ["if", "while", "else"];
+
+function report(atrule, result) {
+  utils.report({
+    message: messages.rejected,
+    node: atrule,
+    result,
+    ruleName
+  });
+}
+
+function fix(atrule) {
+  const regex = /(if)? ?\((.*)\)/;
+
+  // 2 regex groups: 'if ' and cond.
+  const groups = atrule.params.match(regex).slice(1);
+
+  atrule.params = _.uniq(groups).join(" ");
+}
+
+export default function(primary, _, context) {
+  return (root, result) => {
+    const validOptions = utils.validateOptions(result, ruleName, {
+      actual: primary
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    root.walkAtRules(atrule => {
+      // Check if this is a conditional rule.
+      if (!conditional_rules.includes(atrule.name)) {
```

Comment:
This could be changed to use `_.includes`, just in case `includes` would not be supported somewhere.